packages/jaeger-ui/src/components/TracePage/SpanGraph/ViewingLayer.css

```diff
@@ -62,3 +62,15 @@ limitations under the License.
   top: 0;
   user-select: none;
 }
+
+.viewRangeTimeResetButton {
```

Comment:
Please follow the `ViewingLayer--subComponent` naming pattern.

Reply:
will do. I also added a note for the future style guide.

Reply:
renamed the class and a related variable